test-integration/configs/accounts/account1.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
If possible name those accounts such that we have some context what they are used for, i.e. what do they contain and/or which test they serve.

Reply:
done, renamed to non-delegated-clonable-accountX